pyperf/_worker.py

```diff
@@ -310,6 +310,10 @@ def compute(self):
         if args.track_memory:
             if MS_WINDOWS:
                 from pyperf._win_memory import get_peak_pagefile_usage
+            elif MAC_OS:
+                from pyperf._psutil_memory import PeakMemoryUsageThread
+                mem_thread = PeakMemoryUsageThread()
+                mem_thread.start()
             else:
                 from pyperf._memory import PeakMemoryUsageThread
                 mem_thread = PeakMemoryUsageThread()
```

Comment:
It would be interesting to have an abstraction to hide the OS-specific implementation, but if you don't want to do it, you leave it as it is ;-)

Reply:
Let's do it as a seperate issue ;)